### PR TITLE
improved flipper

### PR DESCRIPTION
**Pull request by matthias-luger**
fixes everything

### CODE REVIEW

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-350.flou21.de

File: src/components/Flipper/Flipper.tsx [hunk 1]
```diff
@@ -264,18 +332,29 @@ function Flipper() {
                                     <span style={{ cursor: "pointer" }}><DeleteIcon color="error" /></span>
                                 </span>
                             </Form.Group>
+                            <p>The flipper is stuck <Tooltip type="hover" content={<HelpIcon />} tooltipContent={<span>We get new auctions every 60sek. from Hypixel. So you may have to wait a bit for a new batch to be found.</span>} /></p>
```
Comment (Ekwav) [inline src/components/Flipper/Flipper.tsx:335]: this pushes the flips down to far. Either remove all scroll to top (e.g. AuctionDetails) or make sure the copy button is always visible on full HD screens by squashing filter options (putting them in one line with linewarp)

**Review by Ekwav — CHANGES_REQUESTED** [review]

**Review by Ekwav — APPROVED** [review]
huiiii performant :D

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-350.flou21.de

**Pull request merged**